examples/charts/file/area.py

```diff
@@ -11,11 +14,19 @@
 
 output_file(filename="area.html")
 
-area = Area(
-    xyvalues, title="Responsive Area Chart",
-    xlabel='time', ylabel='memory',
-    stacked=True, legend="top_left",
-    width=400, height=400
+area1 = Area(
+    xyvalues, title="Area Chart",
+    xlabel='time', ylabel='memory', legend="top_left"
+)
+
+area2 = Area(
+    xyvalues, title="Stacked Area Chart", stack=True,
+    xlabel='time', ylabel='memory', legend="top_left"
 )
 
-show(area)
+show(
+    vplot(
+        area1,
+        area2
+    )
+)
```

Comment:
This can all go on one line easily:
```
show(vplot(area1, area2))
```
Many of these go or will go in a gallery so vertical space is not unimportant.